robb1891/robb1891-NewYork.md

```diff
@@ -0,0 +1,6 @@
+## New York
+
+#### Restaurants
+
+#### Things to do
```

Comment:
@robb1891  I love Asian and Indian food.  Are there any sports-related activities available? :basketball: